app/src/main/res/layout/fragment_home.xml

```diff
@@ -1,8 +1,74 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".presentation.home.HomeFragment">
 
-</FrameLayout>
\ No newline at end of file
+    <ImageView
```

Comment:
I would suggest to add the ImageView and TextView for empty state in a separate layout as item_empty, <include /> it in this UI, assign an id and then just show hide easier instead of show hiding both views.